src/Files.App/Helpers/Application/AppLifecycleHelper.cs

```diff
@@ -93,6 +93,8 @@ static Task OptionalTaskAsync(Task task, bool condition)
 
 				return Task.CompletedTask;
 			}
+
+			generalSettingsService.PropertyChanged += GeneralSettingsService_PropertyChanged;
```

Comment:
Will this cause a memory leak if the window is closed and reopened?

Reply:
I believe no. When you have running on background enabled, the process still alive, otherwise GC will finalize when the process is eliminated. FYI @hishitetsu

Reply:
No, this method is called only once at startup.

Reply:
By the way, how do you unhook an event in static class?
For non static class, we should create a event field and dispose it in destructor or Dispose method?

Reply:
Since static classes are not destroyed until the app is exited, the events associated with them should not need to be unsubscribed. If there is a need to unsubscribe in some time, the events should be subscribed/unsubscribed in objects that have another lifecycle.